lib/bundler/remote_specification.rb

```diff
@@ -12,7 +12,7 @@ class RemoteSpecification
 
     attr_reader :name, :version, :platform
     attr_writer :dependencies
-    attr_accessor :source, :remote
```

Comment:
ugh, for some reason I thought this was a subclass, but it's the other way around 🙈